### PR TITLE
[NoTicket] ReactNative HelloWorld app

### PR DESCRIPTION
**Pull request by mvazquez-ms**
## Background

We want/need to gather some knowledge on ReactNative so I'm creating a small PoC to work on it. This is just a simple HelloWorld app, we'll probably add some transitions and a login screen.

## Changes Done

- Added all the required files for a ReactNative HelloWorld app

## Required reviewers

None is _required_, but any feedback is appreciated. I'll add a lot of reviewers, but feel free to ignore this PR if you're busy with something else.

### CODE REVIEW

File: ReactNative/HelloWorld/index.ios.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+export default class HelloWorld extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
```
Comment (briansztamfater) [inline ReactNative/HelloWorld/index.ios.js:12]: Cool! It's like working with HTML & CSS

**Comment by briansztamfater** [comment]
Seems OK to me. Just for curiosity, can you attach a screenshot of the HelloWorld screen running on both iOS Simulator and Genymotion?

Looks like an interesting technology for mobile development!

**Comment by briansztamfater** [comment]
@mvazquez-ms Can we close this one?

**Comment by mvazquez-ms** [comment]
> @mvazquez-ms Can we close this one?

Sure, sorry, I thought this was closed long ago.

**Pull request closed without merging**